Increase timeout to fix route slection issues

stubbsdm opened the pull request:
Some users are still experiencing an issue where the route does not select before the galaxy map shuts. 

This adds extra time on select and hold to try to address the issue.

stubbsdm commented:
Yeh - I've held off doing this until now as I was hoping to avoid it just because It will make the whole experience slower - and the way it was originally written suggested it was meant to be quick - but it would make it reliable.

I'll investigate this approach and see if I can make it feel authentic.

lucaelin commented:
The key is to do both, first attempt the quick one and if we don't receive the feedback go for the slow route. Maybe we can also add some screen reading / template detection for this, to check if route calculation is still in progress.

stubbsdm commented:
OK here's the modified galaxy map macro. I spent a good few hours testing this to get it more reliable and it's looking good on long and short trips, galmap open or closed initially where i found the current macro was not reliable.

Main change is that it now uses wait for condition to check correct completion within a time limit.
But is also:

Changed CamZoomOut to CamZoomIn as zooming in makes the search faster (The camera moves faster to the destination)
Added a check if you already have the route you ask for plotted
changed timings so that longer routes are reliable.
